src/oc_erchef/apps/data_collector/src/data_collector_http.erl

```diff
@@ -0,0 +1,85 @@
+-module(data_collector_http).
```

Comment:
It it mildly troublesome that we now have this same basic code in 4-5 different places (across 3-4 projects).  I think it points to us wanting a different API from opscoderl_httpc.